go/gas-oracle/oracle/base_fee.go

```diff
@@ -0,0 +1,79 @@
+package oracle
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ethereum-optimism/optimism/go/gas-oracle/bindings"
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+func wrapUpdateBaseFee(l1Backend bind.ContractTransactor, l2Backend DeployContractBackend, cfg *Config) (func() error, error) {
```

Comment:
Binding here is interesting. Is this idiomatic Golang? I'm not familiar enough with it to know. I don't mind it, though. Might want to add a comment somewhere that explains why you do this binding.

Reply:
I wrote it this way to make it easier to test. It also follows the pattern used elsewhere in this codebase, for the L2 gas price updating functions